src/app/ui/editor/pixels_movement.cpp

```diff
@@ -112,24 +112,32 @@ PixelsMovement::InnerCmd::MakeStamp(const Transformation& t)
 
 PixelsMovement::PixelsMovement(
   Context* context,
-  Site site,
+  Editor* editor,
   const Image* moveThis,
   const Mask* mask,
   const char* operationName)
   : m_reader(context)
-  , m_site(site)
-  , m_document(site.document())
+  , m_editor(editor)
```

Comment:
It looks like the `Site` parameter was changed by an `Editor` but the editor is being used only to access to the `ColorBar` (which can be already accessed through `ColorBar::instance()`). Not sure if this change is needed.